dapr/proto/common/v1/common.proto

```diff
@@ -104,14 +104,15 @@ message StateOptions {
   // Enum describing the supported concurrency for state.
   enum StateConcurrency {
     CONCURRENCY_UNSPECIFIED = 0;
-    FIRST_WRITE = 1;
-    LAST_WRITE = 2;
+    CONCURRENCY_FIRST_WRITE = 1;
```

Comment:
does it need extra CONCURRENCY_ here

Reply:
this is the [recommended way](https://developers.google.com/protocol-buffers/docs/style#enums). In the generated client codes, we can refer to the enum value like below. 

commonv1pb.StateOptions_CONCURRENCY_LAST_WRITE

In the case that we do not add the prefix, we can refer to the enum value like below,
commonv1pb.StateOptions_LAST_WRITE

Regardless of recommendations, having CURRENCY_ prefix looks more readable to me.

How do you think about it?

cc/ @yaron2

Reply:
Makes sense, sounds good